components/__init__.py

```diff
@@ -242,21 +242,7 @@ def define_components(reqs):
                 libs=['abt'],
                 headers=['abt.h'])
 
-    retriever = GitRepoRetriever("https://review.hpdd.intel.com/coral/cppr",
-                                 True)
-    reqs.define('cppr',
-                retriever=retriever,
-                commands=["scons $JOBS_OPT "
-                          "OMPI_PREBUILT=$OMPI_PREFIX "
-                          "CART_PREBUILT=$CART_PREFIX "
-                          "FUSE_PREBUILT=$FUSE_PREFIX "
-                          "IOF_PREBUILT=$IOF_PREFIX "
-                          "PREFIX=$CPPR_PREFIX install"],
-                headers=["cppr.h"],
-                libs=["cppr"],
-                requires=['iof', 'cart', 'ompi', 'fuse'])
-
-    retriever = GitRepoRetriever("https://review.hppd.intel.com/daos/iof",
+    retriever = GitRepoRetriever("https://github.com/daos-stack/iof",
```

Comment:
While _iof_ is up on github, i'm not sure it's actually being maintained there in any way.

Reply:
Ok, I reverted this one then.   Interestingly, the URL was hppd rather than hpdd so I fixed that too